numpy/core/tests/test_simd.py

```diff
@@ -346,6 +341,31 @@ def test_max(self):
             _max = self.max(vdata_a, vdata_b)
             assert _max == data_max
 
+    def test_min(self):
```

Comment:
the following are missing too
```python
"""
Test intrinics:
    npyv_min_##SFX
    npyv_minp_##SFX
"""
````
I think these changes made during your latest push